hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DefaultSource.scala

```diff
@@ -110,7 +110,8 @@ class DefaultSource extends RelationProvider
     }
     log.info("Obtained hudi table path: " + tablePath)
 
-    val metaClient = HoodieTableMetaClient.builder().setConf(fs.getConf).setBasePath(tablePath).build()
+    val metaClient = HoodieTableMetaClient.builder().setProperties(optParams.asJava)
+      .setConf(fs.getConf).setBasePath(tablePath).build()
```

Comment:
what issue are we fixing with this? this should be separated out as another PR. not mixing it with metaserver bundle  testing setup

Reply:
cause the metaserver enabled is not stored at hoodie.properties, it have to be configured expicity when reading. But the read options aren't transferred to the HoodieMetaClient

Reply:
ok, I submit a pre-pr to fix it?